### DISCUSSION TITLE
Add Apache Age to allowlist of PostgreSQL extensions

### DISCUSSION THREAD

**Discussion by mbledkowski** [prisma/prisma#22403] [Q&A]
Hello,
I would like to ask, to add Apache Age to the allowlist of Postgres extensions.
Apache Age is an extension that adds graph functionality on top of Postgres.
https://github.com/apache/age

**Comment by janpio** [comment]
Does it not being on that list cause you any trouble?

**Reply by mbledkowski** [reply]
I am currently figuring out the implementation of it, without the help of postgresExtensions functionality. I will let you know, if I encounter any roadblocks.

**Reply by janpio** [reply]
I think the only difference is that we do not automatically pick it up on Introspection when it is not part of that list. You can totally put it there manually, and then Prisma will do all the same things and make sure it does not disappear accidentally.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
